sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+- Removed support for Octet Key Pair (OKP) operations.
```

Comment:
I would add a qualifier here to mention that this isn't a breaking change compared to a previous stable version; for example:
> These changes do not impact the API of stable versions such as 4.5.4. Only code written against a beta version such as 4.6.0-beta.1 may be affected.
For example, what we do in Python: https://github.com/Azure/azure-sdk-for-python/blob/4d14d782e507039db31f75686fc44785f734699f/sdk/keyvault/azure-keyvault-keys/CHANGELOG.md#breaking-changes